scripts/zones/Halvung/npcs/qm5.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: Halvung
+--  NPC: ??? (Bracelet of verve)
+-----------------------------------
+local ID = require("scripts/zones/Halvung/IDs"),
+require("scripts/globals/npc_util"),
+require("scripts/globals/keyitems")
+
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHas(trade, 8801) and not player:hasKeyItem(xi.ki.BRACELET_OF_VERVE) then
```

Comment:
Could you add this item ID to itemIDs file (and then convert this to xi.items.ITEM_NAME) or add a coment stating what item it is?

Reply:
Sure

Reply:
Which is the method moving forward? Also where would I add the item to idtemID file? Not familiar with where/what that is.

Reply:
item =
    {
        MOBLIN_OIL             = 17031401,
    },
in IDs.lua?